### DISCUSSION TITLE
Recurring 3 second timer firing in performance tab while app is idle

### DISCUSSION THREAD

**Discussion by dwilt** [vercel/swr#1560] [Q&A]
In an effort to increase performance in our app, we are monitoring the JS thread via the Performance tab in the Google Dev Tools. While the app was idle, I took a Performance recording for 13 seconds and noticed that about every 3 seconds, there was a bunch of processing/activity by SWR. I disabled all of our polling in our app. 

These tasks total up to about 12 milliseconds worth of CPU time but at 6x slowdown that's 72ms every 3 seconds. Any ideas what these might be and is there a way to disable them?

![2021-10-14 19 11 59](https://user-images.githubusercontent.com/1065539/137407664-a29f5add-243f-4f62-ada8-5aeeba9b708f.gif)

**Comment by shuding** [comment]
Thanks for reporting! What's the version of SWR that you're using? Also is there something else that triggers re-rendering periodically?
